assets/js/util/errors.js

```diff
@@ -24,6 +24,8 @@ import isPlainObject from 'lodash/isPlainObject';
 // Export codes and reasons.
 export const ERROR_CODE_MISSING_REQUIRED_SCOPE = 'missing_required_scopes'; // When scopes are missing.
 export const ERROR_REASON_INSUFFICIENT_PERMISSIONS = 'insufficientPermissions';
+export const ERROR_CODE_INSUFFICIENT_MODULE_DEPENDENCIES = 'insufficient_module_dependencies';
+export const ERROR_CODE_ADBLOCKER_ACTIVE = 'adblocker_active';
```

Comment:
I think it'd be more appropriate to define these in the areas where they're used. The first one could go into `assets/js/googlesitekit/modules/datastore/constants.js`, the latter into a new `assets/js/modules/adsense/constants.js`?

Reply:
That was my initial thought, but saw that there was already an established approach and didn't want to deviate from it.

Reply:
Addressed.

Reply:
That's a valid approach. I think the difference is that the error codes currently in there are more general and they're also used by the same general error codebase, while the new ones here are more specific and apply only to another specific area.